src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <IsPackable>true</IsPackable>
+    <!-- Reference the outputs for the dependency nodes calculation. -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <!-- This is a native package and doesn't contain any ref/lib assets. -->
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(NativeBinDir)$(LibPrefix)System.IO.Ports.Native$(LibSuffix)"
```

Comment:
Do you also need to include the symbols?

Reply:
We don't upload runtime specific symbol packages for System.IO.Ports today.

Reply:
Fixed that and verified that the symbol only appears in the symbol package. Please see the added change in runtime.native.System.IO.Ports.props.

Reply:
Also added a diff in the top post for a runtime specific IO.Ports package.